topology/probes/k8s/namespace.go

```diff
@@ -44,25 +44,32 @@ type namespaceProbe struct {
 }
 
 func newObjectIndexer(g *graph.Graph) *graph.MetadataIndexer {
+	ownedByNamespace := []string{
+		"deployment",
+		"daemonset",
+		"ingress",
+		"job",
+		"pod",
+		"persistentvolume",
+		"persistentvolumeclaim",
+		"networkpolicy",
+		"replicaset",
+		"replicationcontroller",
+		"service",
+		"statefulset",
+	}
+
+	ownedByNamespaceFilter := filters.NewOrFilter()
+	for _, ty := range ownedByNamespace {
+		ownedByNamespaceFilter = filters.NewOrFilter(ownedByNamespaceFilter, filters.NewTermStringFilter("Type", ty))
```

Comment:
The new version creates NewOrFilter(NewOrFilter(NewOrFilter(NewOrFilter(term), term), term), term) which is equivalent but create more objects and is not more readable

Reply:
I now only make one OrFilter but I have left code to generate the "Type is equal to" filter from the type of the object (e.g. "pod")